packages/website/src/components/DocContainer.tsx

```diff
@@ -48,11 +49,16 @@ export function DocContainer({ name, kind, excerpt, summary, typeParams, childre
 					</div>
 				) : null}
 				<div className="space-y-10">
-					<Section title="Summary" className="dark:text-white">
-						<p className="text-dark-100 dark:text-gray-300 m-0">{summary ?? 'No summary provided.'}</p>
+					<Section iconElement={<VscListSelection />} title="Summary" className="dark:text-white">
+						<p className="text-dark-100 dark:text-gray-300 m-0 mb-5">{summary ?? 'No summary provided.'}</p>
```

Comment:
You already put `mb-5` on the section itself, putting it here again actually has no effect and it just collapses the margins making them only count as a single `mb-5` instead of 2 `mb-5`